pkg/loader/util.go

```diff
@@ -151,13 +150,16 @@ func CreateDBWithSQLMode(user string, password string, host string, port int, tl
 		}
 		dsn += "&tls=" + name
 	}
+	if params == nil {
+		params = DefaultDBParams
+	}
 
-	return createDBWitSessions(dsn)
+	return createDBWitSessions(dsn, params)
 }
 
 // CreateDB return sql.DB
 func CreateDB(user string, password string, host string, port int, tls *tls.Config) (db *gosql.DB, err error) {
-	return CreateDBWithSQLMode(user, password, host, port, tls, nil)
+	return CreateDBWithSQLMode(user, password, host, port, tls, nil, nil)
```

Comment:
note this change the behavior of `CreateDB` which will set `allow_auto_random_explicit_insert` as `1` before.
```shell
➜  tidb-binlog git:(master) ✗ ag loader.CreateDB
tests/kafka/kafka.go
61:	sinkDB, err := loader.CreateDB("root", "", "127.0.0.1", 3306, nil)

drainer/checkpoint/mysql.go
48:var sqlOpenDB = loader.CreateDB

drainer/relay.go
46:	db, err := loader.CreateDBWithSQLMode(scfg.To.User, scfg.To.Password, scfg.To.Host, scfg.To.Port, scfg.To.TLS, scfg.StrSQLMode)

drainer/sync/mysql.go
45:var createDB = loader.CreateDBWithSQLMode

arbiter/server.go
38:	createDB  = loader.CreateDB

reparo/syncer/mysql.go
50:var createDB = loader.CreateDB
```